### DISCUSSION TITLE
Is it possible to do port-offset connections from the GUI?

### DISCUSSION THREAD

**Discussion by OConnorStP** [jacktrip/jacktrip#973] [Howtos] [answered]
Hi all.  I've got a concert coming up where there will be 3 jacktrip-connected computers on stage so I need to do port-offset connections for two of them.  Not bad with command line, but one of the folks would much rather do it with the GUI.  I thought I'd just check and see if there's a hidden way to do that.

**Comment by psiborg112** [comment, marked as answer]
Are they running as hub clients? All you need to do is change the local port for them to work. (Increase one by one and the other by two. Or to whatever other values you have in mind.)

**Comment by OConnorStP** [comment]
<facepalm>  I've been looking at those port numbers for so long that my eye just skips right over them.  Thanks! </facepalm>
